examples/bpk-component-section-header/examples.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import BpkSectionHeader from '../../packages/bpk-component-section-header';
+import BpkButton from '../../packages/bpk-component-button';
+import SmallLongArrowRightIcon from '../../packages/bpk-component-icon/sm/long-arrow-right';
+import SmallLongArrowLeftIcon from '../../packages/bpk-component-icon/sm/long-arrow-left';
+import { withButtonAlignment } from '../../packages/bpk-component-icon/index';
+import { cssModules, isRTL } from '../../packages/bpk-react-utils';
+
+import STYLES from './examples.module.scss';
+
+const cls = cssModules(STYLES);
```

Comment:
could you rename this to `getClassName` just to align with the rest of the usages